include/dbstructure.php

```diff
@@ -398,6 +398,12 @@ function update_structure($verbose, $action, $tables=null, $definition=null) {
 				$sql3 .= ";";
 			}
 
+			$field_list = '';
```

Comment:
I think this part should only be done when ```$ignore``` was ```false```, since only then the following problematic queries are executed.